ee/tabby-ui/app/files/components/blob-header.tsx

```diff
@@ -120,6 +127,23 @@ export const BlobHeader: React.FC<BlobHeaderProps> = ({
                 <TooltipContent>Download raw file</TooltipContent>
               </Tooltip>
             )}
+            {!enableCodeBrowserQuickActionBar.loading &&
+              enableCodeBrowserQuickActionBar.value &&
```

Comment:
```suggestion
            { enableCodeBrowserQuickActionBar.value &&
```

also need to check that chat playground is enabled.